packages/client/Atmosphere.ts

```diff
@@ -137,11 +137,12 @@ export default class Atmosphere extends Environment {
 
   fetchHTTP = async (body: FetchHTTPData, connectionId?: string) => {
     const uploadables = body.payload.uploadables
-    const headers = {
+    const headers: Record<string, string> = {
```

Comment:
+1
using `as const` on this is slightly stronger as `headers.accept` will be guaranteed to exist

Reply:
done